usecase/parser/authinfo.go

```diff
@@ -5,22 +5,31 @@ import (
 	"strconv"
 
 	cosmosapp_interface "github.com/crypto-com/chain-indexing/appinterface/cosmosapp"
+	applogger "github.com/crypto-com/chain-indexing/external/logger"
 	"github.com/crypto-com/chain-indexing/external/tmcosmosutils"
 	"github.com/crypto-com/chain-indexing/internal/base64"
 	"github.com/crypto-com/chain-indexing/usecase/model"
 	"github.com/crypto-com/chain-indexing/usecase/parser/utils"
 )
 
 func ParseSignerInfosToTransactionSigners(
+	logger applogger.Logger,
 	cosmosClient cosmosapp_interface.Client,
 	signerInfos []utils.SignerInfo,
 	accountAddressPrefix string,
 	possibleSignerAddresses []string,
+	txHash string,
 ) ([]model.TransactionSigner, error) {
 	var signers []model.TransactionSigner
 
 	if len(signerInfos) <= 0 && len(possibleSignerAddresses) <= 0 {
-		panic("error signer info not found")
+		return nil, fmt.Errorf("error signer info not found at tx %q", txHash)
```

Comment:
if returning error here, would un-parsed message type break this? if yes, suggest to log only